draft-ietf-httpbis-sfbis.md

```diff
@@ -464,6 +465,22 @@ For example:
 Example-Date: @1659578233
 ~~~
 
+### Display Strings {#displaystring}
+
+Display Strings are similar to Strings, in that they consist of zero or more characters, but they allow unicode content, unlike Strings.
```

Comment:
```suggestion
Display Strings are similar to Strings, in that they consist of zero or more characters, but the content of a Display String is UTF-8-encoded Unicode, rather than the ASCII carried by a String.
```

Reply:
No - at this level (the data model), they're unicode strings; utf-8 is a characteristic of how they're serialised on the wire.

Reply:
Oh, I see.  This is fine.  Capitalize "Unicode".